specification/loadtestservice/LoadTestService/tspconfig.yaml

```
@@ -0,0 +1,50 @@
+parameters:
+  service-dir:
+    default: "sdk/loadtesting"
+emit:
+  - "@azure-tools/typespec-autorest"
+  # Uncomment this line and add "@azure-tools/typespec-python" to your package.json to generate Python code
+  #- "@azure-tools/typespec-python"
+  #- "@azure-tools/typespec-ts"
+  #- "@azure-tools/typespec-csharp"
+  # Uncomment this line and add "@azure-tools/typespec-java" to your package.json to generate Java code
+  # "@azure-tools/typespec-java": true
+  # Uncomment this line and add "@azure-tools/typespec-csharp" to your package.json to generate C# code
+  # "@azure-tools/typespec-csharp": true
+  # Uncomment this line and add "@azure-tools/typespec-ts" to your package.json to generate Typescript code
+  # "@azure-tools/typespec-ts": true
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    emitter-output-dir: "{project-root}/../"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/loadtestservice.json"
+    azure-resource-provider-folder: "data-plane"
+    examples-directory: ./examples
```

Recommend you configure autorest to omit unreachable types:
```suggestion
    examples-directory: ./examples
    omit-unreachable-types: true
```